projects/netgrif-components-core/src/lib/navigation/navigation-double-drawer/abstract-navigation-double-drawer.ts

```diff
@@ -432,16 +446,22 @@ export abstract class AbstractNavigationDoubleDrawerComponent implements OnInit,
         return roles;
     }
 
-    protected getFilterRoutingPath(filterCase: Case) {
-        const viewTaskId = filterCase.tasks.find(taskPair => taskPair.transition === FILTER_VIEW_TASK_TRANSITION_ID).task;
+    protected getItemRoutingPath(itemCase: Case) {
+        const transId = SETTINGS_TRANSITION_ID;
+        const taskId = itemCase.tasks.find(taskPair => taskPair.transition === transId).task;
         const url = this._dynamicRoutingService.route;
-        return `/${url}/${viewTaskId}`;
+        return `/${url}/${taskId}`;
     }
 
     protected compareStrings(a: string, b: string): number {
```

Comment:
why do we have this method? We should use this instead:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare
it also handles locales. Directions can then be done by simply multiplying the result by `-1` for the reverse order